src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -51,8 +51,6 @@ public class BuildEventArgsReader : IDisposable
             typeof(BuildEventArgs).GetField("threadId", BindingFlags.Instance | BindingFlags.NonPublic);
         private static FieldInfo buildEventArgsFieldSenderName =
             typeof(BuildEventArgs).GetField("senderName", BindingFlags.Instance | BindingFlags.NonPublic);
-        private static FieldInfo buildEventArgsFieldTimestamp =
```

Comment:
This doesn't have to do with the timestamp? That still looks like something that can be written.

Reply:
yes this is separate, was never used, I decided to pile it on